### PR TITLE
Address Critical Missing Information

### PR DESCRIPTION
**Pull request by GloverChris**
This is a proposal to add a notice stating server must have TLS 1.0 enabled. See below for more information.

Symptoms
Assume that you use the inbox Windows Internal Database (WID) in Windows Server. If you disable Transport Layer Security (TLS) 1.0 when you configure security settings, you experience the following issues:
•	The Remote Desktop service (RDS) may fail.
•	An existing RDS deployment that uses Remote Desktop Connection Broker and WID may fail.
•	The Remote Desktop Management service (RDMS) doesn't start.
•	You receive the following error message when you try to start the RDMS:
The Remote Desktop Management service on Local Computer started and then stopped. Some services stop automatically if they are not in use by other services or programs.
•	The Remote Desktop Connection Broker role can't be installed.
•       Server is pending a reboot
•       RemoteDesktop Roles is not installed (Even if it is installed)

Cause
This is expected behavior. This is because of the current dependencies between RDS and Windows Internal Database. RDMS and Connection Broker depend on TLS 1.0 to authenticate with the database. WID doesn't currently support TLS 1.2. So, disabling TLS 1.0 breaks this communication.

Resolution
To fix this issue, use one of the following methods:
•	Set up RDS without Connection Broker for a single-server installation.
•	Don't disable TLS 1.0 on a single Connection Broker deployment.
•	Configure a high availability Connection Broker deployment that uses dedicated SQL Server.

Article Link:
https://docs.microsoft.com/en-us/troubleshoot/windows-server/remote/rds-connection-broker-or-rdms-fails-caused-by-disabled-tls-10

### CODE REVIEW

File: docset/windows/remotedesktop/New-RDSessionDeployment.md
Comment (JohanFreelancer9) [inline docset/windows/remotedesktop/New-RDSessionDeployment.md:33]:
```suggestion
> [!IMPORTANT]
> Transport Layer Security (TLS) version 1.0 must be enabled; if not, the Remote Desktop Session will fail to deploy.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by GloverChris — COMMENTED** [review]
Ah, thanks Johan.

**Comment by GloverChris** [comment]
Committed @JohanFreelancer9 recommendations.

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/windows/remotedesktop/New-RDSessionDeployment.md
Comment (tseward) [inline docset/windows/remotedesktop/New-RDSessionDeployment.md:33]: Because PowerShell itself doesn't support markdown, can you change this to:

Important: \<text\>

**Review by tseward — CHANGES_REQUESTED** [review]

**Review by dariomws — APPROVED** [review]
I approve after @tseward's suggestion.

**Review by get-itips — APPROVED** [review]
I am ok once @tseward suggestions are applied

**Review by GloverChris — COMMENTED** [review]
Added suggestions and included more context in the message as well.

**Comment by GloverChris** [comment]
Thanks @get-itips and @tseward

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by tiburd** [comment]
close > reopen

**Comment by tiburd** [comment]
Hi, @GloverChris and @e0i,

**Following is a blocking issue that needs to be resolved before we can merge:**

docset/windows/remotedesktop/New-RDSessionDeployment.md
- This file is causing a build error:
   Line 0: [Error] Failed to load file: W:/r2lw-s/docset/windows/remotedesktop/New-RDSessionDeployment.md:13:(73) 'Important: Transport Layer Security (TLS) version ...' Expect Heading, please try to use PlatyPS validate the content

***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master)

**Blocking**

- The validation status must be "Passed."

***
Per our admin, @traya1, the warnings in the PR can be ignored for this PR because they are pre-existing.

Thanks!

**Pull request merged**
